yb-voyager/src/version/yb_version.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"golang.org/x/exp/slices"
+)
+
+var supportedYBVersionStableSeriesOld = []string{"2.20"}
+var supportedYBVersionStableSeries = []string{"2024.1"}
+var supportedYBVersionPreviewSeries = []string{"2.21"}
```

Comment:
1. what about the version apart from these?
2. Also if tomorrow 2025 comes, then voyager code will start breaking.. Can we have some generic but smart check, instead of comparing with fixed constants.

Reply:
As discussed, added 2.18, 2.14, 2.23 to this list.